### PR TITLE
Finish mobile version portfolio

### PR DESCRIPTION
**Pull request by lilskyex0x**
# Portfolio: finish mobile version

> This is my personal portfolio for web development ( mobile version )

## Built With

- Html and CSS

## Description of changes

- Finished the mobile version portfolio with exact design from Figma.
- Added required Html and CSS codes and necessary images.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -21,14 +24,186 @@
...
+        <button class="main_btn">Start collaboration</button>
     </section>
+
     <!-- headline section ends -->
+
+
+
+    <!-- work section starts -->
+    <section class="work">
+        <h1 class="title_font projectH1">Projects</h1>
```
Comment (OyePriscilla) [inline index.html]: - [x] Kindly use another header tag for the title. `h1` tag is mostly use to give a clear indication of the content of the page. 😄

File: index.html [hunk 2]
```diff
@@ -21,14 +24,186 @@
...
+                        <li><button class="btn2">HTML/CSS</button></li>
+                        <li><button class="btn2">Ruby on Rails</button></li>
+                        <li><button class="btn2">JavaScript</button></li>
+                    </ul>
+                    <div class="btn3">
+                       <h2 class="text">See this project</h2>
+                       <img class="arrow" src="img/ic_arrow_right.png">
+                    </div>
+                </div>
+            </div>
```
Comment (OyePriscilla) [inline index.html]: - [x] Kindly use `section` tag to wrap the content of all the cards. It has more semantics that div. 👍 You can as well check the requirement for confirmation.

File: index.html [hunk 3]
```diff
@@ -21,14 +24,186 @@
...
+                <img class="project_img" src="img/Project2.png" alt="Project1">
+                <div class="menu">
+                    <h2 class="sub_title">Project name goes here</h2>
+                    <ul class="list">
+                        <li><button class="btn2">HTML/CSS</button></li>
+                        <li><button class="btn2">Ruby on Rails</button></li>
+                        <li><button class="btn2">JavaScript</button></li>
+                    </ul>
+                    <div class="btn3">
+                       <h2 class="text">See this project</h2>
```
Comment (OyePriscilla) [inline index.html]: - [x] This element suppose to be a link a or button tag, and kindly follow the button styling: Active, Hover and Pressed for all the cards. 😄

File: index.html [hunk 4]
```diff
@@ -21,14 +24,186 @@
...
+            </div>
+        </div>
+    </section>
+    <!-- work section ends -->
+
+
+    <!-- about section starts -->
+    <section class="about">
+        <div class="cv_block">
+            <h1 class="title_font">About</h1>
```
Comment (OyePriscilla) [inline index.html]: - [x] Kindly change this to another header tag different from `h1`.  `h1` tag is mostly use to give a clear indication of the content of the page. 😄

File: style.css [hunk 5]
```diff
@@ -79,4 +81,224 @@ body {
...
-/* ------------------------------------------------------- */
+/* -------------------Work Section CSS-------------------------------- */
+
+.work {
+  position: absolute;
+  width: 320px;
+  height: 2134px;
+  left: 0;
+  top: 672px;
+}
```
Comment (OyePriscilla) [inline style.css]: - [x] Kindly display your work section with grid according to the requirement. 👍

File: style.css [hunk 6]
```diff
@@ -79,4 +81,224 @@ body {
...
+.showcase {
+  width: 320px;
+  height: 2052px;
+  margin-top: 82px;
+}
+
+.project {
+  width: 320px;
+  height: 342px;
+}
```
Comment (OyePriscilla) [inline style.css]: - [x] Give your project display of flex as well according to the requirement. 👍

**Review by OyePriscilla — CHANGES_REQUESTED** [review]
## Hi @lilskyex0x  😄 ,
Honestly, you've done a great work 🤝, but there are few issues to rectify. Be encouraged, you're almost there. 👍 


 
## (Highlights) Good Points: 
* No linter errors. :heavy_check_mark:
* Followed Github flow :heavy_check_mark:
* PR has a good title and summary :heavy_check_mark:
* Commit messages are meaningful :heavy_check_mark:

## (Changes Required) 

* [x]  Please check my inline comments for other changes required.
 
## [Optional] suggestions:
 
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
 
**Cheers and Happy coding! :clap::clap::clap:**
 
Feel free to leave any questions or comments in the PR thread if something is not 100% clear
Please, remember to tag @oyepriscilla in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
 
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by frank1738 — APPROVED** [review]
Hi @lilskyex0x , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights

✅ Correct use of Github workflow
✅ No linter error
✅ README is okay


## Optional suggestions

N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**